### PR TITLE
update name and logos

### PR DESCRIPTION
**Pull request by lukavdplas**
Updates name and logos for our lab name change.

Notes:

- I have no idea which permutation of RSL / CDH / UU should be used
- This uses the CDH logo in the footer, which I think is what we decided on?

### CODE REVIEW

File: {{cookiecutter.slug}}/frontend.angular/src/app/footer/footer.component.html [hunk 1]
```diff
@@ -13,13 +13,9 @@ <h1 class="subtitle" i18n>{{cookiecutter.project_title}}</h1>
...
-                <div class="column has-text-right-tablet">
-                    <a class="university-logo image" href="https://uu.nl" target="_blank">
-                        <img src="{{"{{environment.assets}}"}}/utrecht-university.svg" alt="Utrecht University logo" i18n-alt width="200"
-                            height="50" />
-                    </a>
-                    <a class="lab-logo image" href="https://dig.hum.uu.nl" target="_blank">
-                        <img src="{{"{{environment.assets}}"}}/dhlab.svg" alt="Digital Humanities Lab logo" i18n-alt width="200" height="40" />
+                <div class="column has-text-right-tablet is-narrow">
+                    <a class="lab-logo image" href="https://cdh.uu.nl/centre-for-digital-humanities/research-software-lab/" target="_blank">
+                        <img src="{{"{{environment.assets}}"}}/uu-cdh.png" alt="Centre for Digital Humanities, Utrecht University" i18n-alt width="200" height="40" />
```
Comment (jgonggrijp) [inline {{cookiecutter.slug}}/frontend.angular/src/app/footer/footer.component.html]: Would an SVG version be feasible? Nicer for high resolution displays.

**Review by jgonggrijp — COMMENTED** [review]
Can we use an SVG image and host it at dhstatic?

File: {{cookiecutter.slug}}/frontend.angular/src/app/footer/footer.component.html (hunk 1 above)
Comment (lukavdplas) [reply {{cookiecutter.slug}}/frontend.angular/src/app/footer/footer.component.html]: Good one. There are no SVGs in the [branding](https://github.com/CentreForDigitalHumanities/Branding) repository, I'll ask if we can get one.

Comment (lukavdplas) [reply {{cookiecutter.slug}}/frontend.angular/src/app/footer/footer.component.html]: Done! :D

**Pull request merged**